packages/client-graphql/src/messages/index.ts

```diff
@@ -2,30 +2,28 @@ import { Client } from "urql";
 import { ICourierClientParams } from "../types";
 import { createCourierClient } from "../client";
 
-export const GET_UNREAD_MESSAGE_COUNT = `
+export const GET_MESSAGE_COUNT = `
   query MessageCount($params: FilterParamsInput) {
     messageCount(params: $params)
   }
 `;
 
 export interface IMessageCountParams {
-  isRead?: boolean;
+  tags?: string[];
   from?: number;
+  isRead?: boolean;
 }
-type GetUnreadMessageCount = (params?: IMessageCountParams) => Promise<number>;
-export const getUnreadMessageCount = (
-  client?: Client
-): GetUnreadMessageCount => async (params) => {
+type GetMessageCount = (params?: IMessageCountParams) => Promise<number>;
+export const getMessageCount = (client?: Client): GetMessageCount => async (
```

Comment:
nobody is importing this one directly yet so i can change it.  i verified with color